packages/mask/src/extension/options-page/DashboardComponents/CollectibleList/index.tsx

```diff
@@ -301,7 +301,7 @@ export function CollectionList({
                                 className={classes.button}
                                 variant="outlined"
                                 size="small">
-                                {formatEthereumAddress(addressName.label, 5)}
+                                <ReverseAddress address={addressName.resolvedAddress} addressSize={5} />
```

Comment:
suggest, `5` value as address size by default, don't repeat yourself.

Reply:
suggest, rename the component name to `ReversedAddress`

the is react component, should be used when done.

e.g: `formatDate` as component, `FormattedDate`

Reply:
resolved in #5914